pyiceberg/typedef.py

```diff
@@ -74,8 +74,14 @@ def __missing__(self, key: K) -> V:
 
 
 Identifier = Tuple[str, ...]
+"""Type for identifiers in PyIceberg."""
```

Comment:
Since the hyperlink will show this docstring, lets add something more descriptive. 
Perhaps something like
```
 Identifier = Tuple[str, ...]
"""A tuple of strings representing a table identifier.

Each string in the tuple represents a part of the table's unique path. For example, 
a table in a namespace might be identified as:

    ("namespace", "table_name")

Examples:
    >>> identifier: Identifier = ("namespace", "table_name")
"""
```

Reply:
OK, I added this.